src/kv.rs

```diff
@@ -107,20 +92,51 @@ async fn add_claim(claims: &mut Vec<HashMap<String, String>>, data: Option<Vec<u
     Ok(())
 }
 
-// NOTE: these tests require nats to be running with JS enabled. To run these, explicitly run them as they
-// are difficult to run from within CI
+pub(crate) fn ld_hash(ld: &LinkDefinition) -> String {
+    ld_hash_raw(&ld.actor_id, &ld.contract_id, &ld.link_name)
+}
+
+pub(crate) fn ld_hash_raw(actor_id: &str, contract_id: &str, link_name: &str) -> String {
```

Comment:
Does this work exactly the same as https://github.com/wasmCloud/wasmcloud-otp/pull/520/files ?

If so, I think each location should have a comment referencing the other repo's file, so if we ever update one place we remember to update the other as well